scripts/build_platform_config.py

```diff
@@ -266,7 +268,7 @@ def codeOutDevicePin(device, pin, definition_name):
   #codeOut("#define JSVAR_CACHE_SIZE                "+str(200)+" // Number of JavaScript variables in RAM")
 else:
   codeOut("#define JSVAR_CACHE_SIZE                "+str(variables)+" // Number of JavaScript variables in RAM")
-  codeOut("#define FLASH_AVAILABLE_FOR_CODE        "+str(flash_available_for_code))
+  codeOut("#define FLASH_AVAILABLE_FOR_CODE        "+str(int(flash_available_for_code)))
```

Comment:
My python produced outputs like 1024.0 for these defines without explicitly casting them to ints  so I guess there's a different default for this.
On a side note, is flash_saved_code_pages in the same file supposed to be an int or a float? Leaving it as a float has the side effect of the MAGIC words not being word-aligned..